File: app/src/main/java/de/nils_beyer/android/Vertretungen/LicenseActivity.java
package de.nils_beyer.android.Vertretungen;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.Toolbar;
import android.view.View;

import de.nils_beyer.android.Vertretungen.R;

public class LicenseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_license);

        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        getSupportActionBar().setTitle(getString(R.string.info_title_libraries));

        toolbar.setNavigationIcon(R.drawable.ic_keyboard_arrow_left_white_48dp);
        toolbar.setNavigationOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();

            }
        });
        toolbar.setContentInsetStartWithNavigation(0);
        getSupportActionBar().setHomeButtonEnabled(true);
    }
}

File: app/src/main/java/de/nils_beyer/android/Vertretungen/LicenseActivity.kt
package de.nils_beyer.android.Vertretungen

import kotlinx.android.synthetic.main.activity_license.*

import android.support.v7.app.AppCompatActivity
import android.os.Bundle


class LicenseActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_license)

        // Setup Toolbar
        setSupportActionBar(toolbar)
        supportActionBar!!.title = getString(R.string.info_title_libraries)

        // Setup navigation button
        toolbar.setNavigationIcon(R.drawable.ic_keyboard_arrow_left_white_48dp)
        toolbar.setNavigationOnClickListener { finish() }
        toolbar.contentInsetStartWithNavigation = 0
        supportActionBar!!.setHomeButtonEnabled(true)
    }
}
